tests/test_schema.py

```diff
@@ -4029,6 +4029,18 @@ def test_schema_get_migration_union_ptrs_01(self):
 
         self._assert_migration_consistency(schema)
 
+    def test_schema_get_migration_union_ptrs_02(self):
+        schema = r'''
+        type A;
+        type A_ extending A;
+        type B;
+        type C {
+            link foo -> A | A_ | B;
+        };
+        '''
+
+        self._assert_migration_consistency(schema)
```

Comment:
Could you reframe this as being one of the `self._assert_migration_equivalence(` tests, and testing that it works to *remove* a link with a type like this?